docs/terraform_loadtest.md

```diff
@@ -0,0 +1,75 @@
+# Running a load-test with a Terraform deployment
+
+This guide describes how to setup and run a load-test using a [Terraform](https://www.terraform.io/intro/index.html) deployment.  
+Following this process it is possible to create a complete [load-test system](loadtest_system.md) in a mostly automated form.  
+This is the recommended way to load-test a Mattermost instance for production.
+
+## Prerequisites
+
+- [Terraform](https://learn.hashicorp.com/terraform/getting-started/install)
+- AWS credentials to be used as described [here](https://www.terraform.io/docs/providers/aws/index.html#authentication)
+- A Mattermost E20 license, required to run the load-test through the [`coordinator`](coordinator.md)
+
+**Note**
+
+If authenticating using the [AWS credentials file](https://www.terraform.io/docs/providers/aws/index.html#shared-credentials-file), the profile to use is `mm-loadtest`.
+
+### Clone the repository
+
+```sh
+git clone https://github.com/mattermost/mattermost-load-test-ng
+cd mattermost-load-test-ng
```

Comment:
I think we can move `cd mattermost-load-test-ng` below. It is not related to cloning the repository.

Reply:
I actually wasn't sure whether to include it in the first place but thought that maybe there will be some less command line experienced users that might need it.